homeassistant/components/fronius/sensor.py

```diff
@@ -483,20 +517,23 @@ async def async_setup_entry(
         native_unit_of_measurement=ENERGY_WATT_HOUR,
         device_class=DEVICE_CLASS_ENERGY,
         state_class=STATE_CLASS_TOTAL_INCREASING,
+        entity_registry_enabled_default=False,
     ),
     SensorEntityDescription(
         key="energy_year",
         name="Energy year",
         native_unit_of_measurement=ENERGY_WATT_HOUR,
         device_class=DEVICE_CLASS_ENERGY,
         state_class=STATE_CLASS_TOTAL_INCREASING,
+        entity_registry_enabled_default=False,
     ),
     SensorEntityDescription(
         key="energy_total",
         name="Energy total",
         native_unit_of_measurement=ENERGY_WATT_HOUR,
         device_class=DEVICE_CLASS_ENERGY,
         state_class=STATE_CLASS_TOTAL_INCREASING,
+        entity_registry_enabled_default=False,
```

Comment:
This sounds like a entity that would be used with the energy dashboard... should that be disabled by default?

Reply:
This is the cumulative energy of all inverters. I think its a better choice to use the total energy of every individual inverter for the energy dashboard.
I've read in community forums that this cumulative value sometimes can vary (on older multi-inverter systems) because it doesn't account for one inverter sometimes. That's why I chose to disable by default.

Reply:
Alright! Thanks! 👍